Lombiq.NodeJs.Extensions/pnpm-lock.yaml

```diff
@@ -0,0 +1,5193 @@
+lockfileVersion: 5.3
```

Comment:
Why do we need this now and didn't need it before?

Reply:
I doubt it wasn't needed before. If you run `pnpm install` (which the _Lombiq.NodeJs.Extensions.targets_ does) then this file is created.

Reply:
Hmm. It wasn't here before, wasn't created, and seemingly didn't cause any issues. But I don't understand why it was missing before.